apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/db/BlockDaoImpl.java

```diff
@@ -644,14 +627,7 @@ public void deleteAll() {
                 stmt.executeUpdate("SET REFERENTIAL_INTEGRITY FALSE");
```

Comment:
Probably we don't need that command any more, do we?
Anyway it's minor issue.

Reply:
Fixed